### ISSUE TITLE
Update the screenshot for the expected output in the main readme

### ISSUE THREAD

**Issue by allyhawkins** [AlexsLemonade/scpca-downstream-analyses#241]
The[ current screenshot](https://github.com/AlexsLemonade/scpca-downstream-analyses/blob/main/screenshots/expected_output_structure.png) of the expected output structure is a cutoff image, we should take a new screenshot that doesn't include the lines on the side.

**Comment by jaclyn-taroni** [comment]
I don't think it needs to be a screenshot. It will probably be much easier to maintain if it's a code block instead.

**Comment by cbethell** [comment]
Addressed by merged PR #251
